integration/analyzer_peliasQuery.js

```diff
@@ -49,6 +49,16 @@ module.exports.tests.functional = function(test, common){
     assertAnalysis( 'place', 'Toys "R" Us!', [ 'toys', 'r', 'us' ]);
     assertAnalysis( 'address', '101 mapzen place', [ '101', 'mapzen', 'place' ]);
 
+     // complicated tokenization for some Asian languages
+     assertAnalysis('thai_address1', 'ซอยเพชรบุรี๑', ['ซอย', 'เพชรบุรี1'] );
```

Comment:
Can you explain a little more about the segmentation here?

Are these token groups from research/knowledge or just a copy->paste of what was generated?

The number 1 on the end catches my eye and I'm curious how it appeared from `ซอยเพชรบุรี๑`

```
'ซอย', 'เพชรบุรี1'
```

Reply:
>Are these token groups from research/knowledge or just a copy->paste of what was generated?

Well, I used ChatGPT to generate them and then tools like Google Translate to validate, that these tokens indeed have separate meanings... Tbh it would be great to have someone native to one of those languages to validate 🤔  I will try to find one 😄 

>The number 1 on the end catches my eye and I'm curious how it appeared from ซอยเพชรบุรี๑

Thai language has its own digits (but they use Arabic ones as well btw) and `๑` is `1`. Btw we convert them already even without this tokenizer. Not sure at which step though. But good question is if this number should be a separate token, but IIRC it is how it works right now(if you input `foo1` it will generate single `foo1` token), so I decided to limit  scope of changes and may be will address it with separate PR. WDYT?

Reply:
and yeah, we should get a human to check a few examples, maybe beat to generate a few more before asking someone in different classes such as address, venue(s), streets, intersections, localities etc.

Reply:
this feature will go through a bunch of testing which might take a long time, flagging potential issues now I'll be much easier than finding them later, I'm fairly confident we can resolve any issues.